lib/AppInfo/Application.php

```diff
@@ -191,17 +196,22 @@ protected function registerCollaborationResources() {
 	}
 
 	public function registerFullTextSearch() {
-		if (Util::getVersion()[0] < 16 || !\OC::$server->getAppManager()->isEnabledForUser('fulltextsearch')) {
+		if (Util::getVersion()[0] < 16) {
 			return;
 		}
 
 		$c = $this->getContainer();
 		try {
 			$this->fullTextSearchService = $c->query(FullTextSearchService::class);
+			$this->fullTextSearchManager = $c->query(FullTextSearchManager::class);
```

Comment:
Should be IFullTextSearchManager, FullTextSearchManager is not part of the public API

Reply:
Yeah, I changed like 3 times before getting it wrong...